client/components/web-preview/style.scss

```diff
@@ -141,12 +141,6 @@ a.web-preview__external.button {
 	display: flex;
 	flex-wrap: wrap;
```

Comment:
Is there any reason why the toolbar should have a "flex-wrap: wrap"? It seems unlikely we ever want buttons to wrap, no?